lib/cdo/contact_rollups.rb

```diff
@@ -682,7 +675,7 @@ def self.update_professional_learning_attendance_for_course_from_sections(course
 
   def self.mysql_multi_connection
     # return a connection with the MULTI_STATEMENTS flag set that allows multiple statements in one DB call
-    Sequel.connect(CDO.pegasus_reporting_db_writer.sub('mysql:', 'mysql2:'), flags: ::Mysql2::Client::MULTI_STATEMENTS)
+    Sequel.connect(CDO.PEGASUS_DB_WRITER.sub('mysql:', 'mysql2:'), flags: ::Mysql2::Client::MULTI_STATEMENTS)
```

Comment:
This looks like a possible mistake from the find-and-replace?

Reply:
Yes.  And would have been caught by the simple integration test that @uponthesun suggested to me this afternoon.  Fixing ...

Reply:
fixed